ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml

```diff
@@ -26,6 +26,15 @@ images:
 - dockerfile_path: openshift-ci/Dockerfile.registry.build
   from: operator-registry-base
   to: performance-addon-operator-registry
+- dockerfile_path: openshift-ci/Dockerfile.registry.upstream.dev
+  from: operator-registry-base
+  to: performance-addon-operator-registry-master
+promotion:
+  excluded_images:
+  - operator-registry-base
+  - performance-addon-operator-registry
+  name: "performance-addon-operator"
+  namespace: ocp
```

Comment:
Should we also have a `tag` ?

Reply:
Also, what will happen to `performance-addon-operator-registry-master`? It isn't either promoted explicitly nor excluded.